### ISSUE TITLE
Saving as .py file doesn't comment out raw cells

### ISSUE THREAD

**Issue by phaabe** [jupyter/notebook#4109]
When saving a jupyter notebook as a .py file the raw cells appear as normal code and are not commented out. I believe that's an issue. 

The raw cells should be commented out.

**Comment by NexYY** [comment]
Agree. Using markdown instead of raw is also not an option, since it will render every commend as level 1 heading.
